.changeset/soft-bobcats-learn.md

```diff
@@ -0,0 +1,6 @@
+---
+'@hono/standard-validator': patch
```

Comment:
It looks like "fix", but it's better to set a new feature. So can you replace `patch` with `minor`?

Reply:
Hooks are exported, and I think that changing their type parameters is a breaking change.

Reply:
So, you mean it should be `major`?